webui/react/src/components/SlotAllocationBar.tsx

```diff
@@ -148,6 +160,29 @@ const SlotAllocationBar: React.FC<Props> = ({
           <Bar {...barProps} parts={barParts} />
         </div>
       </ConditionalWrapper>
+      {footer && (
+        <div className={css.footer}>
+          {poolType === V1ResourcePoolType.K8S ? (
+            <header>{`${footer.isAux ?
+              `${footer.auxRunning} Aux Container Running` :
```

Comment:
In this and line 173 using `footer.auxRunning`, should be <number> Aux Container**s** Running